lib/Runtime/Language/WasmAsmJsCommon.h

```diff
@@ -0,0 +1,9 @@
+//-------------------------------------------------------------------------------------------------------
+// Copyright (C) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE.txt file in the project root for full license information.
+// Portions of this file are copyright 2014 Mozilla Foundation, available under the Apache 2.0 license.
+//-------------------------------------------------------------------------------------------------------
+
+#pragma once
+
+#include "WAsmJsUtils.h"
```

Comment:
I was planning to have multiple files (upcoming PR), but I think I will use only this one. Will make the change